sled-agent/src/hardware/illumos/gpt.rs

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! A thin wrapper of interfaces to the GPT.
+//!
+//! Enables either real or faked GPT access.
+
+use libefi_illumos::Error;
+use std::path::Path;
+
+/// Trait to sub-in for access to the libefi_illumos::Gpt.
+///
+/// Feel free to extend this interface to exactly match the methods exposed
+/// by libefi_illumos::Gpt for testing.
+pub(crate) trait LibEFIGpt {
```

Comment:
Style nit - `LibEfiGpt` maybe (similarly `LibEfiPartition`)?

Reply:
Sure, I'm always on the fence with acronyms (sometimes, the letters *should* be capitalized!), but this updated to `LibEfi...`.